### PR TITLE
Review tables for dbcreatedate and dbupdatedate

### PR DESCRIPTION
**Pull request by coverbeck**
**Description**
TLDR; this PR adds dbcreatedate to one table. :) 

***Details***

The ticket was to review all tables to ensure they had dbcreatedate and dbupdatedate, and that both fields were populating correctly in all those tables. Table-by-table details are in SEAB-3083, but broadly this is what I discovered:

* I couldn't add the `@UpdateTimestamp` annotation to the dbupdatedate for `@Embeddable` entities. Dockstore just wouldn't start up. See [here](https://en.wikibooks.org/wiki/Java_Persistence/ElementCollection). I initially had a commit that removed the dbupdatedate column property for embeddables, but then I realized that could force us to have downtime when we deploy 1.15; if we remove the column during the 1.15 deploy, and 1.14 is still serving requests, 1.14 would error.
* Join tables don't have dbcreate nor dbupdate columns, and I couldn't figure out a way to add them. And I don't know that it's worth it.
* `@ElementCollection` also have issues with the two timestamp annotations.
* After looking at every table, I believe the only changed needed was to add dbcreatedate to one entity/table, fullworkflowpath. That table is an outlier, populated by a trigger and doesn't do updates, just inserts, so I left off dbupdatedate.

**Review Instructions**
1. Create a new workflow one day in qa
2. The next day, download the QA DB, and verify that the dbcreatedate column is set for the new row in the table `fullworkflowpath`.

**Issue**
SEAB-3083

**Security and Privacy**

None

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that you pass the basic style checks and unit tests by running `mvn clean install`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] Follow the existing JPA patterns for queries, using named parameters, to avoid SQL injection
- [x] If you are changing dependencies, check the Snyk status check or the dashboard to ensure you are not introducing new high/critical vulnerabilities
- [x] Assume that inputs to the API can be malicious, and sanitize and/or check for Denial of Service type values, e.g., massive sizes
- [x] Do not serve user-uploaded binary images through the Dockstore API
- [x] Ensure that endpoints that only allow privileged access enforce that with the `@RolesAllowed` annotation
- [x] Do not create cookies, although this may change in the future
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.

### CODE REVIEW

**Comment by kathy-t** [comment]
> I initially had a commit that removed the dbupdatedate column property for embeddables, but then I realized that could force us to have downtime when we deploy 1.15; if we remove the column during the 1.15 deploy, and 1.14 is still serving requests, 1.14 would error.

FYI, we will already have downtime for 1.15 https://github.com/dockstore/dockstore-deploy/wiki/1.15-staging#steps-applicable-for-this-specific-release-1

**Review by kathy-t — APPROVED** [review]

**Comment by denis-yuen** [comment]
Hmmm, the original table with the issue is enduser, but it doesn't seem to have the issue either which is good.

**Comment by denis-yuen** [comment]
> Join tables don't have dbcreate nor dbupdate columns, and I couldn't figure out a way to add them. And I don't know that it's worth it.

Would be nice, but seems not worth it if we have to resort to something like a trigger

**Review by denis-yuen — APPROVED** [review]

**Comment by coverbeck** [comment]
> > I initially had a commit that removed the dbupdatedate column property for embeddables, but then I realized that could force us to have downtime when we deploy 1.15; if we remove the column during the 1.15 deploy, and 1.14 is still serving requests, 1.14 would error.
> 
> FYI, we will already have downtime for 1.15 https://github.com/dockstore/dockstore-deploy/wiki/1.15-staging#steps-applicable-for-this-specific-release-1

Ah, good point, I'd forgotten that. I should try to remove them then.

**Comment by coverbeck** [comment]
Removed unused dbupdatedate columns, re-requesting reviews.

**Comment by denis-yuen** [comment]
FYI, gotta fill out that security section :wink:

**Review by denis-yuen — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]
